src/FINDOLOGIC/Api/Requester.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace FINDOLOGIC\Api;
+
+use FINDOLOGIC\Api\Exceptions\ConfigException;
+use FINDOLOGIC\Api\RequestBuilders\Json\SuggestionRequestBuilder;
+use FINDOLOGIC\Api\RequestBuilders\RequestBuilder;
+use FINDOLOGIC\Api\RequestBuilders\Xml\NavigationRequestBuilder;
+use FINDOLOGIC\Api\RequestBuilders\Xml\SearchRequestBuilder;
+use InvalidArgumentException;
+
+abstract class Requester
```

Comment:
Odd class name. I'm sure there can be a better one.

Reply:
Something cool and sleek, but I am not very imaginative regarding such things.

Reply:
Alternatively, you could just make the one method this class has a method of `RequestBuilder`, which would IMO be more intuitive anyway.